src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -360,7 +375,7 @@ private void runOnlineReplay(int replayId) {
         });
   }
 
-  private void runFafReplayFile(Path path) throws IOException {
+  private void runFafReplayFile(Path path, String overrideMapName) throws IOException {
```

Comment:
Please tell me why is that needed and please make a some java doc /comment on it because it does not seem very obvious